.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Pull request recommendations:
+
+- [ ] Name your pull request `your-development-type/short-description`.
+
+Ex:
+> feature/read-tiff-files
+
+or
+
+> bugfix/handle-file-not-found
+
+- [ ] If your pull request addresses a GitHub issue, link to it either in a commit message or in the PR description.
+
+Ex:
+> Resolves [gh-12], adds tiff file format support to data loader
+
+- [ ] Provide context of changes regardless of if reviewers have intimate knowledge of the repository.
```

Comment:
I would just have this say `Provide context of changes`

Reply:
I just updated both the pull request and issue templates because I am actually upgrading all of cookiecutter. Now we will have those fancy schmancy issues "bug report" or "feature request" templates for issues and a pr template. Check the changes when you can!